test/e2e/density.go

```diff
@@ -210,6 +214,7 @@ var _ = Describe("Density", func() {
 				PodStatusFile:        fileHndl,
 				Replicas:             totalPods,
 				MaxContainerFailures: &MaxContainerFailures,
+				Timeout:              time.Duration(math.Ceil(float64(itArg.podsPerNode)/minPodStartupThroughput)) * time.Minute,
```

Comment:
Sure. I got this from https://github.com/kubernetes/kubernetes/blob/f23034d5dac2be305b419db55dd344fbb8738f5d/test/e2e/metrics_util.go#L449